lib/assets/gallery-video-article.mst

```diff
@@ -62,6 +62,8 @@ let article_content = document.getElementById('article-content');
 if (article_content != null) {
     if (article_content.children.length < 1) {
```

Comment:
I know it's not directly part of this PR but these two ifs should be combined:
`if (article_content && article_content.children.length) {`